src/infohud.asm

```diff
@@ -521,6 +521,7 @@ ih_hud_code:
     JSR (.status_display_table,X)
 
     ; Samus' HP
+    LDA #$0C0E : STA $7EC690 ; erase stale decimal tile
```

Comment:
I think it's okay to do this after the comparison; you might see the decimal after changing categories but only until HP changes.  I think we should avoid affecting lag wherever possible.

Reply:
Wait you already changed it.  What?  How are you reading my mind???

Reply:
It was blocking my % for CPU Usage :)